### PR TITLE
Adding food items  pictures routes 

### PR DESCRIPTION
**Pull request by chiragasawa**
Fixes Issue No. #60

### CODE REVIEW

File: src/api/routes/foodphotos.ts
Comment (paramsiddharth) [inline src/api/routes/foodphotos.ts:37]: The endpoint should be consistent with the format of the `/photos` endpoint. Have a look at [this](https://github.com/adityabisoi/REST-fetch/pull/70).

The response format is:
```js
{
	"data": [
		... // The objects
	]
}
```

What your code would return is:
```js
[
	... // The objects
]
```

So what you need to do is:
```js
res.status(200).json({ data: data.data });
```

This update is needed to ensure consistency with the `/photos` endpoint, since `/photos/food` is an extension to the same.

File: src/api/routes/foodphotos.ts
Comment (paramsiddharth) [inline src/api/routes/foodphotos.ts:47]: The same update is needed as above.

**Review by paramsiddharth — CHANGES_REQUESTED** [review]

**Comment by chiragasawa** [comment]
api response has been updated according to requirements.

**Comment by paramsiddharth** [comment]
> api response has been updated according to requirements.

@chiragasawa Looks great to me! I'll merge it after #64 is merged to avoid conflicts.

**Comment by chiragasawa** [comment]
@paramsiddharth please merge this i want to add more features i may need this code in that.

**Comment by paramsiddharth** [comment]
@chiragasawa Excellent work. Merged!

**Review by paramsiddharth — APPROVED** [review]

**Pull request merged**
